frontend/src/utils/color.ts

```diff
@@ -0,0 +1,16 @@
+// Copied from https://stackoverflow.com/a/59387478
+export const lighten = (color: string, lighten: number) => {
```

Comment:
We can add a little documentation string, to precise what is expected in `color` and `lighten`. 

Example, with a non-documented `lighten` parameter.

```suggestion
/**
 * Return a lighten version of an hexadecimal color.
 *
 * Adapted from: Copied from https://stackoverflow.com/a/59387478
 * 
 * @param color An hexadecimal color to lighten.
 * @param lighten A positive integer? number? representing...
 * @returns An hexadecimal lighten version of the original `color`. 
 */
 export const lighten = (color: string, lighten: number): string => {
```

Reply:
Done in 4325c03ea6a76d404b7153c711f827cc3bd626d4.